README.md

```diff
@@ -1,149 +1,158 @@
-
-### CLI
-
-`hellofx` [sample](HelloFX/CLI): simple HelloFX class to run on command line, without build tools
-
-### Maven
-
-`hellofx` [sample](HelloFX/Maven): simple HelloFX class to run with Maven
-
-### Gradle
-
-`hellofx` [sample](HelloFX/Gradle): simple HelloFX class to run with Gradle
-
-## CommandLine
 
-Contains samples of modular and non-modular projects that can be run from command line, with or without build tools.
+Description
+---
 
-### Non-Modular
+This repository contains a collection of HelloFX samples. Each one is a very simple 
+HelloWorld sample created with JavaFX 11 that can be run with different options and build tools.
 
-#### CLI
-
-`hellofx` [sample](CommandLine/Non-modular/CLI): non-modular project to run on command line, without build tools
-
-#### Maven
-
-`hellofx` [sample](CommandLine/Non-modular/Maven): non-modular project to run with Maven
-
-#### Gradle
-
-`hellofx` [sample](CommandLine/Non-modular/Gradle): non-modular project to run with Gradle
-
-### Modular
-
-#### CLI
-
-`hellofx` [sample](CommandLine/Modular/CLI): modular project to run on command line, without build tools
-
-#### Maven
-
-`hellofx` [sample](CommandLine/Modular/Maven): modular project to run with Maven
+The related documentation for each sample can be found [here](https://openjfx.io/openjfx-docs/).
 
-#### Gradle
+For more information go to https://openjfx.io.
 
-`hellofx` [sample](CommandLine/Modular/Gradle): modular project to run with Gradle
 
-## IDEs
 
-Contains samples of modular and non-modular projects that can be run from an IDE, with or without build tools.
+Content
+---
 
-### IntelliJ
+* [HelloFX samples](#HelloFX-Samples)
+* [CommandLine](#CommandLine)
+    - [_Non-modular samples_](#CLI-Non-Modular-Samples)
+    - [_Modular samples_](#CLI-Modular-Samples)
+* [IDEs](#IDEs)
+    - [IntelliJ](#IntelliJ)  
+      [_Non-modular samples_](#IntelliJ-Non-Modular-Samples)  
+      [_Modular samples_](#IntelliJ-Modular-Samples)  
+    - [NetBeans](#NetBeans)  
+      [_Non-modular samples_](#NetBeans-Non-Modular-Samples)   
+      [_Modular samples_](#NetBeans-Modular-Samples) 
+    - [Eclipse](#Eclipse)  
+      [_Non-modular samples_](#Eclipse-Non-Modular-Samples)   
+      [_Modular samples_](#Eclipse-Modular-Samples)
+* [License](#License)
+* [Contribution](#Contribution)
 
-#### Non-Modular
 
-##### Java
 
-`hellofx` [sample](IDE/IntelliJ/Non-Modular/Java): non-modular project to run from IntelliJ
+HelloFX samples<a name="HelloFX-Samples" />
+---
 
-##### Maven
+Contains samples of a simple HelloFX class that can be run from command line, with 
+or without build tools.
 
-`hellofx` [sample](IDE/IntelliJ/Non-Modular/Maven): non-modular project to run from IntelliJ, with Maven
+Build Tools | Sample | Description
+----- | ------- | -----------
+Java | [HelloFX project](HelloFX/CLI) | Simple HelloFX class to run on command line, without build tools.
```

Comment:
Should we say `None` for the build tool instead of `Java`? You could then remove `, without build tools` from the description.

Reply:
Update the first two sections 'HelloFX samples' and 'Command Line'.

What I can say is that in NetBeans 8.2 (momentary I haven't time to install the NetBeans 9) if I compile a normal Java(FX) project then 'Ant' (build.xml) will be used per default. So in case from NetBeans we can write 'Ant'. Or do I misunderstood something?
What is in the other IDEs the case? How will Eclipse, IntelliJ compile a 'normal' project? With Ant or None or...?

Reply:
I believe all of them use Ant. On command line we don't use it of course. Initially we were using Ant as category for IDEs, but we went to Java after that, as the IDE is the one that manages internally (usually via build.xml files) how to call the java commands.